features/cellular/framework/targets/UBLOX/N2XX/UBLOX_N2XX_CellularNetwork.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2018, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "UBLOX_N2XX_CellularNetwork.h"
+
+using namespace mbed;
+
+UBLOX_N2XX_CellularNetwork::UBLOX_N2XX_CellularNetwork(ATHandler &atHandler) : AT_CellularNetwork(atHandler)
+{
+    _op_act = RAT_UNKNOWN;
+}
+
+UBLOX_N2XX_CellularNetwork::~UBLOX_N2XX_CellularNetwork()
+{
+    if (_connection_status_cb) {
+        _connection_status_cb(NSAPI_EVENT_CONNECTION_STATUS_CHANGE, NSAPI_ERROR_CONNECTION_LOST);
+    }
+}
+
+AT_CellularNetwork::RegistrationMode UBLOX_N2XX_CellularNetwork::has_registration(RegistrationType reg_type)
+{
+    return (reg_type == C_EREG) ? RegistrationModeLAC : RegistrationModeDisable;
+}
+
+nsapi_error_t UBLOX_N2XX_CellularNetwork::set_access_technology_impl(RadioAccessTechnology opRat) // Not understandable, RAT (+CEDRXS)
+{
+    switch (opRat) {
+#if defined(TARGET_UBLOX_C030_U201) || defined(TARGET_UBLOX_C027)
```

Comment:
Do we need the non-N2xx related #defines in here?

Reply:
yes, no need to check for others.